test/integration/framework/src/siftool/test_utils.py

```diff
@@ -358,12 +359,18 @@ def get_blocklist_sc(self):
         result = self.w3_conn.eth.contract(address=address, abi=abi)
         return result
 
-    def get_bridge_bank_sc(self):
+    def get_bridge_bank_sc(self) -> Contract:
```

Comment:
Can our typings be more specific then Contract? Like BridgeBankContract? I assume this Contract object gives access to methods unique to BridgeBank?

Reply:
The web3.eth method returns a Base contract object, which has a `Caller` object you can use for something like this 
```    
contract = ctx.get_bridge_bank_sc()
output = contract.caller.hasBlocklist()
```
This code runs but
Type hint is not available because the functions are read from abi and added as part of `__init__`